runtime/jcl/common/jclcinit.c

```diff
@@ -170,28 +175,32 @@ jint computeFullVersionString(J9JavaVM* vm)
 	strcat(fullversion, aotEnabled ? "en" : "dis");
 	strcat(vminfo, "abled, AOT ");
 	strcat(vminfo, aotEnabled ? "en" : "dis");
-	strcat(fullversion, "abled)\nJ9VM - ");
-	strcat(vminfo, "abled)\nJ9VM - ");
+	strcat(fullversion, "abled)\n");
+	strcat(fullversion, OPENJ9_REPO);
+	strcat(vminfo, "abled)\n");
+	strcat(vminfo, OPENJ9_REPO);
```

Comment:
These pairs of calls to `strcat` can be merged by expressing the second arguments as:
`"abled)\n" OPENJ9_REPO`

Reply:
Updated in latest.